lib/index.js

```diff
@@ -11,7 +12,17 @@ async function beforeRunHook({ config }) {
     return;
   }
 
-  const { outputDir, jsonDir, reporterOptions } = await setConfig(config);
+  const simpleConfig = await setConfig(config);
+  const { outputDir, jsonDir, reporterOptions, configOutput } = simpleConfig;
+
+  if (configOutput) {
+    debugLog(`Write simple config to ${configOutput}`);
+    // create folder if not exist
+    await fse.mkdirp(path.dirname(configOutput));
+
+    // write simple config to file
+    await fse.writeJSON(configOutput, simpleConfig, { spaces: 2 });
+  }
 
   // Only if overwrite is set to false delete only .jsons folder and keep the html files
   // By default overwrite is set to undefined (which means true by mochawesome-report-generator)
```

Comment:
`.jsons` folder will be removed before each thread starts, which will cause many json reports missing.

Reply:
IMHO, `overwrite` option should only overwrite individual report file. And add a new option `cleanOutputBeforeRun` to remove the output folder(including .jsons folder which users do not care) before each run.

Reply:
Hey (:
I took care of the overwrite option [here](https://github.com/LironEr/cypress-mochawesome-reporter/pull/155/files#diff-d5109920b34c2b2e141bd41232d8876432a17479c388e142d8a036bb7b36ea62R95)

When using cypress-parallel I can't delete the output folder before the run because when this reporter is called it's already in a thread, That's why I changed the overwrite flag to false and you must clean the output folder [manually](https://github.com/LironEr/cypress-mochawesome-reporter/pull/155/files#diff-a9c1f214fbef5a43cfa249ddb4d0782e6914bfe13a61bd31316e9138a013f5d0R26) before the run

Reply:
When overwrite option is false, the output folder is not deleted, that's correct. But the `.jsons` folder is deleted:
```
  if (reporterOptions.overwrite === false) {
    log(`Remove .jsons folder ${jsonDir}`);
    await fse.remove(jsonDir);
  }
```
That mean when you start a new thread, previous generated json files will be deleted, and they will be missing when merging.

Reply:
Oh got it, Yeah you are right.

it worked until now because all the threads start at the same time and probably start to write JSON files only after all threads delete the .jsons folder.. anyway, I should fix that

thanks

Reply:
LGTM